Makefile

```diff
@@ -49,7 +49,7 @@ build.docker: build.linux
 	docker build --rm -t "$(IMAGE):$(TAG)" -f $(DOCKERFILE) --build-arg TARGETARCH= .
 
 build.docker.multiarch: build.linux.amd64 build.linux.arm64
-	docker buildx create --use
+	docker buildx create --config /etc/cdp-buildkitd.toml --driver-opt network=host --bootstrap --use
```

Comment:
I think we should move the whole `docker buildx` stuff to the internal build because this is not relevant in the open source repo, only in the CDP build.

Reply:
Okay, I'll remove the `multiarch` target 😄

Reply:
Done.